docs/guides/cases/connect-unsupported-protocol.md

```diff
@@ -1,10 +1,199 @@
 ---
-title: Connect Devices with Unsupported Protocols
+title: Access devices with unsupported protocols
 sidebar_position: 5
 ---
 
-# Connect Devices with Unsupported Protocols
+:::note
+Although ***Shifu*** has been unremittingly trying to support more protocols, there are so many protocols for IoT device communication that it is inevitable to meet with protocols not yet supported when using ***Shifu***. This article will show you how to access devices in ***Shifu*** using third-party drivers.
+:::
+
+:::tip
+If you'd like to contribute codes and add a new driver to ***Shifu*** project, please check [***deviceShifu*** Development Guide](https://github.com/Edgenesis/shifu/blob/main/docs/development/develop-deviceshifu.md).
+:::
+
+## Technical Principle
+
+You can check the folder [pkg/deviceshifu](https://github.com/Edgenesis/shifu/tree/main/pkg/) in [***Shifu*** repository](https://github.com/Edgenesis/shifu.deviceshifu) to see the ***deviceShifu***s supported by ***Shifu***. You can see that [pkg/deviceshifu/deviceshifuhttp](https://github.com/Edgenesis/shifu/tree/main/pkg/deviceshifu/deviceshifuhttp) will convert an IoT device using HTTP protocol to ***debiveShifu***.
+In other words, we only need to convert unsupported protocols to HTTP interfaces.
+
+## Use Case
+
+Take a device with RS485-Modbus protocol as an example.
+
+:::note
+[RS485](https://baike.baidu.com/item/RS-485/9695794) is a hardware connection protocol, and [Modbus](https://baike.baidu.com/item/Modbus通讯协议/5972462) is a software layer connection protocol.
```

Comment:
replace with https://en.wikipedia.org/wiki/RS-485 and https://en.wikipedia.org/wiki/Modbus